libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -464,7 +463,33 @@ namespace impl {
          mvo("context_free_data", ptrx.get_context_free_data());
          mvo("packed_trx", ptrx.get_packed_transaction());
          add(mvo, "transaction", trx, resolver, ctx);
+         out(name, std::move(mvo));
+      }
 
+      /**
+       * overload of to_variant_object for packed_transaction, providing original packed_transaction_v0 variant layout
+       *
+       * This matches the FC_REFLECT for this type, but this is provided to allow extracting the contents of ptrx.transaction
+       */
+      template<typename Resolver>
+      static void add( mutable_variant_object &out, const char* name, const packed_transaction& ptrx, Resolver resolver, abi_traverse_context& ctx )
+      {
+         static_assert(fc::reflector<packed_transaction>::total_member_count == 3);
+         auto h = ctx.enter_scope();
+         mutable_variant_object mvo;
+         auto trx = ptrx.get_transaction();
```

Comment:
`const auto&`

Reply:
Updated `signed_transaction`, `transaction`, `transaction_header` to disallow implicit copy. Removed copies we do not need.